### ISSUE TITLE
Why there is no speaker labels provided with the pretrained model Titanet large from NGC cloud? 

### ISSUE THREAD

**Issue by mujhenahiata** [NVIDIA/NeMo#8820]
this is not a bug but if i want to hide the speaker labels, how can i do that for my finetuned model.  @nithinraok  @titu1994 @meghmak13

**Comment by nithinraok** [comment]
`model.labels=None`?

**Comment by mujhenahiata** [comment]
Yes
